src/components/App/App.tsx

```diff
@@ -18,7 +18,7 @@ import { Player } from "../../utils/player";
 const App = () => {
   const [players, setPlayers] = useState(
     //version local storage key to prevent stale data with app changes
-    JSON.parse(localStorage.getItem("game_v1.0.0")) ||
+    JSON.parse(localStorage.getItem("game") || "") || //change to game_v1.0.0
```

Comment:
Probably want to take care of the TODO item, now, to prevent issues when deploying?